README.md

```diff
@@ -59,8 +119,8 @@ to `./runner.yml` and overriding relevant properties.
 
 To set up the project run:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
+./vendor/bin/run drupal:site-setup
```

Comment:
This is already run as post command of composer.

Reply:
The all paragraph makes sens, see:

> Before setting up and installing the site make sure to customize default configuration values by copying [runner.yml.dist](runner.yml.dist)
> to `./runner.yml` and overriding relevant properties.
> 
> To set up the project run:
> 
> ```bash
> ./vendor/bin/run drupal:site-setup
> ```